first-party/jib-layer-filter-extension-maven/README.md

```diff
@@ -0,0 +1,47 @@
+# Jib Layer-Filter Extension
+
+A general-purpose layer-filter extension that enables fine-grained layer control, including deleting files and moving files into new layers. Note that the extension can only filters files put by Jib; it cannot filter files in the base image.
+
+## Examples
+
+```xml
+<pluginExtension>
+  <implementation>com.google.cloud.tools.jib.maven.extension.layerfilter.JibLayerFilterExtension</implementation>
+  <configuration implementation="com.google.cloud.tools.jib.maven.extension.layerfilter.Configuration">
+    <filters>
+      <!-- Delete all jar files. -->
```

Comment:
This confuses me, especially given that we say: 

> If multiple filters match a file, the last filter in the order applies to the file.

does this end up being a no-op is all jars are matched below?

Reply:
Updated to "Delete all jar files (unless they match the filters below)."

So, only deleted will be the JARs not matching the filters below.